### PR TITLE
use postgresql instead of h2

### PR DESCRIPTION
**Pull request by antromeo**

### CODE REVIEW

**Review by zonia3000 — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -151,17 +151,17 @@
             </activation>
```
Comment (cecchisandrone) [inline pom.xml]: Just remove the profiles and always include the same dependency

File: src/main/resources/config/application.yaml [hunk 2]
```diff
@@ -33,18 +33,18 @@ app:
...
-#       ddl-auto: update
\ No newline at end of file
+spring:
+   datasource:
+     # Enable this section for postgresql
+     driver-class-name: org.postgresql.Driver
+     url: jdbc:postgresql://localhost:5432/strapi-general-config-db
+     username: strapi-user
+     password: strapi-password
+     # url: jdbc:postgresql://10.0.91.139:5432/strapi_general_config_db
```
Comment (cecchisandrone) [inline src/main/resources/config/application.yaml]: remove this

File: ui/widgets/widgets-dir/strapi-config-app/.env [hunk 3]
```diff
@@ -11,11 +11,11 @@ PORT=3001
 # REACT_APP_PUBLIC_URL=http://172.40.0.142:8081/api/config/
 
 ######------- Local ENV -----------
-# REACT_APP_KEYCLOAK_URL=http://20.79.198.96.nip.io/auth
```
Comment (cecchisandrone) [inline ui/widgets/widgets-dir/strapi-config-app/.env:14]: remove commented code if not used. If necessary, move this to a different .env file

File: ui/widgets/widgets-dir/strapi-config-app/.env [hunk 4]
```diff
@@ -11,11 +11,11 @@ PORT=3001
...
+# REACT_APP_KEYCLOAK_URL=http://localhost:9080/auth
 # REACT_APP_KEYCLOAK_REALM=entando
 # REACT_APP_KEYCLOAK_CLIENT_ID=web_app
 
 ######------- Integration ENV -----------
-REACT_APP_PUBLIC_URL=/entando-strapi-config-api/api/config/
+REACT_APP_PUBLIC_URL=http://localhost:8081/api/config/
```
Comment (cecchisandrone) [inline ui/widgets/widgets-dir/strapi-config-app/.env]: Did you test this when deployed on Entando platform?

**Review by cecchisandrone — APPROVED** [review]

**Pull request merged**